nodes/src/nodes/telegram/IEndpoint.py

```
@@ -0,0 +1,425 @@
+
+        The engine strips the field namespace prefix ('telegram.') before
+        storing values, so keys arrive flat: 'botToken', 'mode', 'webhookUrl'.
+        """
+        try:
+            parameters = self.endpoint.serviceConfig['parameters']
+            return parameters
+        except Exception as e:
+            debug(f'Telegram _get_telegram_config: EXCEPTION {e}')
+            return {}
+
+    async def _startup(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_startup`

Add return type annotation: `None`

(ANN202)